pom.xml

```diff
@@ -25,11 +25,11 @@
   <parent>
     <groupId>com.xwiki.parent</groupId>
     <artifactId>xwikisas-parent-platform</artifactId>
-    <version>14.10-2</version>
+    <version>14.10-1</version>
```

Comment:
Why decrease the parent version?

Reply:
The upgrade should be done to version 14.10, and the associated version tag is `14.10-1`: https://github.com/xwikisas/xwikisas-parent/blob/xwikisas-parent-14.10-1/xwikisas-parent-platform/pom.xml#L32

Reply:
There's an important distinction between dash ``-`` and dot ``.`` when talking about parent version:
* 14.10.2 means all XWiki dependencies will be from version 14.10.2
* 14.10-2 means all XWiki dependencies will be from version **14.10**; the ``-2`` suffix indicates that this is the second version of this parent POM
* 14.10.X-Y in general means this is the Y iteration of the parent pom that brings XWiki 14.10.X dependencies

So you should upgrade the parent to ``14.10-2`` which is currently the latest iteration of the 14.10 parent pom.